schema.sql

```diff
@@ -8,5 +9,19 @@ CREATE TABLE animals (
   weight_kg DECIMAL
 );
 
--- Add new column species -- 
-ALTER TABLE animals ADD COLUMN species CHAR(40);
\ No newline at end of file
+-- Create the owners table --
+CREATE TABLE owners (
+  id SERIAL PRIMARY KEY,
+  full_name VARCHAR(50),
+  age INT
+);
+
+-- Create the species table --
+CREATE TABLE species (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(50)
+);
+
+-- Modify the animals table to add foreign key references --
+ALTER TABLE animals ADD COLUMN species_id INT REFERENCES species(id);
+ALTER TABLE animals ADD COLUMN owner_id INT REFERENCES owners(id);
```

Comment:
- [ ] Great job working on all the queries so far but I think you forgot to remove the column as mentioned in the requirements, it would be great if you remove column species from the animal table so you can alter the table and drop the column ⭐⭐💯


![image](https://github.com/GhausAliKhan/veternity-clinic/assets/58553711/c82b1703-7236-4b6f-914a-20db16f7bc53)